mono/metadata/sgen-tarjan-bridge.c

```diff
@@ -398,6 +439,13 @@ dyn_array_ptr_contains (DynPtrArray *da, void *x)
 }
 
 static gboolean
+match_colors_estimate (DynPtrArray *a, DynPtrArray *b)
+{
+	return dyn_array_ptr_size (a) == dyn_array_ptr_size (b);
+}
+
+
+static gboolean
 match_colors (DynPtrArray *a, DynPtrArray *b)
```

Comment:
Ugh, I was not paying attention and misunderstood what the hash function was doing. Let me take a look at this...